src/Navplace.php

```diff
@@ -17,15 +17,15 @@ public function __construct($mods, $url)
 
         $this->data = $mods;
         $this->url = $url;
-        $this->coordinates = $mods->query('subject[@authority="geonames"]/cartographics/coordinates');
```

Comment:
I don't think I understand why this is being removed. But I think this is just because I'm unfamiliar with this.